applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -254,6 +284,11 @@ static int initialize(le_audio_receive_cb recv_cb)
 
 int le_audio_config_get(uint32_t *bitrate, uint32_t *sampling_rate)
 {
+	int frames_per_sec = 1000000 / bt_codec_cfg_get_frame_duration_us(streams.codec);
```

Comment:
So if I understand correctly, "streams" is not an array since currently only one stream is supported?

Reply:
Yes, this part is tricky, since for now it only support one stream.